tools/nuget/Desktop/Microsoft.MixedReality.WebRTC.Native.Desktop.props

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemDefinitionGroup>
+        <ClCompile>
+            <LanguageStandard>stdcpp14</LanguageStandard>
```

Comment:
What does this do?

Reply:
Taken offline: good catch, not needed since we now expose a pure C API so are not relying on any C++ standard feature.

Reply:
Fixed.